examples/example_diffev.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+"""
+Example using the Student's t log-likelihood
+for robust fitting of data with outliers.
```

Comment:
The `resid` function looks like a normal residual. I can't see why this is using Student's t log-likelihood.